pages/common/zopflipng.md

```diff
@@ -0,0 +1,15 @@
+# zopflipng
+
+> PNG image compression utility.
+
+- Optimize a PNG image:
+
+`zopflipng {{input.png}} {{output.png}}`
+
+- Optimize and overwrite several PNG images:
```

Comment:
So how do I decide when will it generate an output image and when will it overwrite ? According to your example, I cannot have a case where I want to overwrite 2 images. Because if there are 2 images one will be the output image. And if there are 3, then all of them will be overwritten.

Can you clarify this ?

Reply:
I think I made a mistake. That format does not work. I'll remove that code.